src/FasterDBStore/FasterDBStore.csproj

```diff
@@ -0,0 +1,22 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.1</TargetFramework>
+    <RootNamespace>Neo.Plugins.Storage</RootNamespace>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <None Update="RocksDBStore\config.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+      <CopyToPublishDirectory>PreserveNewest</CopyToPublishDirectory>
+    </None>
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.FASTER" Version="2019.11.18.1" />
+    <PackageReference Include="Mono.Posix.NETStandard" Version="1.0.0" />
```

Comment:
What's this?

Reply:
Without this, travis doesn't work... I don't know why

Reply:
https://github.com/neo-project/neo-plugins/runs/338134148